adoptability.md

```diff
@@ -53,14 +55,20 @@ external systems (e.g. third-party software, web portals, and other
 internal DHIS2 modules) to securely access and manipulate DHIS2 data. The DPG 
 MOSIP also publishes [good documentation on its APIs](https://docs.mosip.io/platform/apis). 
 
+[TODO: note that the above section is the summary of key recommendations + 
```

Comment:
Thanks.  I will try to do something in the final version to make that separation clear in every module.

Reply:
ok

Reply:
ok. I removed the TODO.

Reply:
ok - good luck :)